src/main/java/org/candlepin/subscriptions/security/SecurityConfig.java

```diff
@@ -124,6 +123,17 @@ protected void configure(HttpSecurity http) throws Exception {
                 // Allow access to Actuator endpoints here
                 .requestMatchers(EndpointRequest.to("health", "info", "prometheus")).permitAll()
                 .antMatchers("/**/openapi.*", "/**/version", "/api-docs/**", "/webjars/**").permitAll()
+                // ingress security is done via server settings (require ssl cert auth), so permit all here
+                .antMatchers("/api/rhsm-subscriptions/v1/ingress/**").permitAll()
```

Comment:
`api` and `rhsm-subscriptions` can technically vary based on the `PATH_PREFIX` and `APP_NAME` environment variables.  If we want to keep offering those values as being configurable,  we should pull those values from the Spring Environment (maybe inject them as `@Value`?).

Reply:
DONE